<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuidLaserSetting</class>
 <widget class="QWidget" name="GuidLaserSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,8">
   <property name="topMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>This device is optional, please add it according to the actual requirement, select the quantity you need after adding, do not need the device, please choose not to add it and click next.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QRadioButton" name="radioButton_set">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_ignore">
       <property name="text">
        <string>Do not Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vLayout_content"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
